atlasdb-server/src/main/java/com/palantir/atlasdb/server/config/ClientConfig.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.server.config;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.palantir.atlasdb.spi.KeyValueServiceConfig;
+
+@JsonSerialize(as = ImmutableClientConfig.class)
+@JsonDeserialize(as = ImmutableClientConfig.class)
+@Value.Immutable
+public abstract class ClientConfig {
+    public abstract String client();
```

Comment:
Not easy to do and I'm not sure we even care about the keyspace being the same. It shouldn't really be enforced to be a 1:1 mapping.